sys/net/network_layer/ng_ipv6/netif/ng_ipv6_netif.c

```diff
@@ -38,49 +38,58 @@ static char addr_str[NG_IPV6_ADDR_MAX_STR_LEN];
 static ng_ipv6_addr_t *_add_addr_to_entry(ng_ipv6_netif_t *entry, const ng_ipv6_addr_t *addr,
                                           uint8_t prefix_len, uint8_t flags)
 {
+    ng_ipv6_netif_addr_t *free_entry;
```

Comment:
Of course, you're right. Just blurted out words without thinking :D. My point stands however: `free_entry` is confusing in the context of the function name. Maybe `free_addr` to keep it in the context of the later?